frame/contracts/src/wasm/runtime.rs

```diff
@@ -945,7 +1052,7 @@ define_env!(Env, <E: Ext>,
 	// `SENTINEL` is returned as a sentinel value.
 	[seal0] seal_contains_storage(ctx, key_ptr: u32) -> u32 => {
```

Comment:
Should probably be factored out to its own function on `Runtime` just as for `seal_set_storage`.